app/policies/organization_policy.rb

```diff
@@ -30,8 +30,4 @@ def admin_of_org?
   def generate_new_secret?
     update?
   end
-
-  def pro_org_user?
-    user.pro? && OrganizationMembership.exists?(user_id: user.id, organization_id: record.id)
-  end
```

Comment:
This was [only ever used](https://github.com/forem/forem/pull/13156/files#diff-8176f7cf6d68ad6630418f9d9faf7be510e5fb6e5e0bb535fe36efe3ed36bcb5L61) by the `DashboardsController#pro` action, so we can remove this now! ✂️ ✅